### ISSUE TITLE
support for async matchers

### ISSUE THREAD

**Issue by bertBruynooghe** [tlvince/eslint-plugin-jasmine#169]
**Feature request**

**Current behavior?**

e.g. `await expectAsync(new Promise...` triggers `'expectAsync' is not defined`

**What is the expected behavior?**
At least it should pass wit

**Please mention your node.js, eslint-plugin-jasmine and operating system version.**
Node: 9
eslint-plugin-jasmine: 2.10.1
os: MacOS, docker node:9-alpine
jasmine 3.2.0

**Comment by sidvishnoi** [comment]
I think this would be handled by eslint globals instead.

**Comment by bertBruynooghe** [comment]
@sidvishnoi Can you please elaborate? Is there a way we could it get working now?

**Comment by sidvishnoi** [comment]
This project's readme says:
> ESLint itself provides a Jasmine environment for Jasmine's global variables

meaning globals are not handled here.

So, we can add to following to `.eslintrc.yml`:
``` yml
globals:
  expectAsync: true
``` 

It would be a workaround until  https://github.com/sindresorhus/globals/pull/152 is merged and included in eslint

**Comment by bertBruynooghe** [comment]
ok, thx, makes sense.
